src/main/java/frc/robot/commands/GoToPose.java

```diff
@@ -0,0 +1,67 @@
+package frc.robot.commands;
+
+import java.util.function.Supplier;
+
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.math.geometry.Transform2d;
+import edu.wpi.first.math.util.Units;
+import edu.wpi.first.wpilibj.Timer;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.constants.TestConstants;
+import frc.robot.subsystems.Drivetrain;
+
+/**
+ * Tests the odometry of the robot by driving a certain distance and calculating the error.
+ */
+public class GoToPose extends CommandBase {
+
+  private Drivetrain m_drive; 
+  private Supplier<Pose2d> m_poseSupplier;
+  private double m_startTime;
+  private Pose2d m_finalPose;
+  private Pose2d m_error;
+  
+  public GoToPose(Drivetrain drive, Pose2d pose) {
+    m_drive = drive; 
+    // finalPose is position after robot moves from current position-- startPose-- by the values that are inputted-- distanceToMove
+    m_finalPose = pose;
+    addRequirements(drive);
```

Comment:
Could just do m_poseSupplier = () -> pose
That would make the pose supplier a "supplier" but it always supplies the same pose. Just cleaner so variables aren't null

Reply:
Also the comment is unclear